scripts/plotting/hgdp_1kg_tob_wgs_pca_new_variants_nfe_no_outliers/hgdp_1kg_tob_wgs_plot_pca_nfe.py

```diff
@@ -33,7 +33,13 @@ def query():
         | (mt.s.contains('TOB'))
     )
     # remove outlier samples
-    mt = mt.filter_cols((mt.s != 'TOB1734') & (mt.s != 'TOB1714') & (mt.s != 'TOB1126'))
+    mt = mt.filter_cols(
```

Comment:
Could you instead do a `semi_join_cols` with the `scores` table?

Reply:
nice, that's a great idea. In that case, I could also remove the line where I filter for NFE and TOB samples only